pkg/operator/ceph/config/defaults.go

```diff
@@ -67,8 +92,9 @@ func DefaultLegacyConfigs() *Config {
 		Set("osd pool default pg num", "100").
 		Set("osd pool default pgp num", "100").
 		//
-		Set("rbd_default_features", "3"). // TODO: still needed?
+		Set("rbd_default_features", "3") // . // TODO: still needed?
 		//
-		Set("fatal signal handlers", "false") // TODO: still needed?
+		// this makes mds fail ... ???
+		// Set("fatal signal handlers", "false") // TODO: still needed?
```

Comment:
still experimenting with the settings in this section?

Reply:
Oh. This was removed sort of by mistake. I found fatal signal handlers to be useful for giving extra error messages (even though they weren't very accurate) when the mdses were failing. But it's **very** verbose, and I imagine that might be why it was set in the first place. Thoughts?

Reply:
I think I should leave this as it was for now. I'll take care of that.